apps/expo-go/ios/Exponent/Versioned/Core/Api/EXUtil.m

```diff
@@ -38,6 +38,17 @@ + (NSString *)escapedResourceName:(NSString *)name
   return [name stringByAddingPercentEncodingWithAllowedCharacters:allowedCharacters];
 }
 
++ (BOOL)isExcludedExpoHost:(NSString *)host
```

Comment:
If preferred, we could invert this logic and instead only _include_ `u.expo.dev`; I wasn't sure if there were any other subdomains.